template/prettier.config.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2020, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = require('@sumup/foundry/prettier')(
+  {},
+  {
+    quoteProps: 'consistent',
```

Comment:
This is already part of the [default config](https://github.com/sumup-oss/foundry/blob/main/src/configs/prettier/config.ts#L32).

Reply:
👏